diff --git a/tools/services/user_manager.py b/tools/services/user_manager.py
index dbde361..aa66159 100644
--- a/tools/services/user_manager.py
+++ b/tools/services/user_manager.py
@@ -109,7 +109,7 @@ NoDisplay={str(hide).lower()}
     global stopping
     stopping = False
     args.user_manager = threading.Thread(target=service_thread)
-    args.user_manager.start()
+    #args.user_manager.start()
 
 def stop(args):
     global stopping
